astropy/units/quantity.py

```diff
@@ -1139,6 +1127,26 @@ def _repr_latex_(self):
         lstr
             A LaTeX string with the contents of this Quantity
         """
+        if unit is not None and unit != self.unit:
+            return self.to(unit).to_string(
+                unit=None, precision=precision, format=format, subfmt=subfmt)
+
+        formats = {
+            None: None,
+            "latex": {
+                None: ("$", "$"),
+                "inline": ("$", "$"),
+                "display": (r"$$", r"$$"),
```

Comment:
Does a notebook work with `$\displaystyle`? If not, one other suggestion is to call this "$$" or "jupyter". Or actually keep it as `display`, but set it to `$\displaystyle` and worry about `jupyter` in the next PR, when we do `_repr_latex_`.